src/deploy/osp_deployer/settings_sanity.py

```diff
@@ -128,6 +135,25 @@ def check_ipmi_to_nodes(self):
                                         self.settings.ipmi_user,
                                         self.settings.ipmi_password)
 
+    def check_mtu_of_networks(self):
+        shouldbbevalidmtu = [
+            'mtu_size_global_default',
+            'public_api_network_mtu',
+            'private_api_network_mtu',
+            'storage_network_mtu',
+            'provisioning_network_mtu',
+            'storage_cluster_network_mtu',
+            'tenant_tunnel_network_mtu',
+            'floating_ip_network_mtu',
+            'management_network_mtu',
+            'tenant_network_mtu',
+        ]
+        for each in shouldbbevalidmtu:
+            assert self.is_valid_mtu(getattr(self.settings, each)),\
+                "Setting for " + each + " is not a valid mtu " +\
+                getattr(self.settings, each) + "Maximum mtu size allowed = "\
+                "9216 and minimum allowed =594"
+
```

Comment:
if the range is already enforced in the profiles json, you don't need the above.